answers/networking.md

```diff
@@ -164,7 +164,9 @@ To be defined.
 
 ## TCP or UDP will be used in this case? why?
 
-To be defined.
+TCP: used in web browsing, email, file transfers,... because it is a reliable protocol (requires handshake), guarantee the data integrity. 
```

Comment:
Hi @hieubz . Thank your for your contribution.
About your answser, I'm afraid that you misundertood the question.
I believe in this question, `"this case"` refers to the above question `## The other of place to look up DNS.`.